spec/cerner/oauth1a/access_token_spec.rb

```diff
@@ -6,15 +6,282 @@
 
 # rubocop:disable Metrics/BlockLength
 RSpec.describe Cerner::OAuth1a::AccessToken do
+  describe '.from_authorization_header' do
+    context 'raises error' do
```

Comment:
This is more stylistic, but I feel like the context and specs are reversed. Looking at [BetterSpecs](http://www.betterspecs.org/), they recommend using contexts for describing certain conditions and usually the actual tests are of the form `it 'does something' do` or `it 'should do sometihng' do`

Reply:
Can you give me some solid examples of how this would be rewritten?

To be frank, I'm exhausted with the "style of specs", which I fully believe NO ONE CAN DO CORRECTLY, and am about two seconds away from throwing RSpec out the window and never using it again.